R/pull.R

```diff
@@ -75,6 +75,9 @@ NULL
 #' @rdname workflow-extractors
```

Comment:
Similar to workflowsets, it would be nice to add the `lifecycle::badge("soft-deprecated")` badge to the doc files of each of these

Reply:
AFAICT there is no "soft-deprecated" badge. `usethis::use_lifecycle(https://lifecycle.r-lib.org/articles/stages.html)` doesn't make one and [our website]() doesn't show one.  

`lifecycle::badge("deprecated")` works though. 

Admittedly, the documentation in `?badge` makes you think that you can use it.

Reply:
Do you have lifecycle 1.0.0? There definitely is one:

``` r
lifecycle::badge("soft-deprecated")
#> [1] "\\ifelse{html}{\\href{https://lifecycle.r-lib.org/articles/stages.html#soft-deprecated}{\\figure{lifecycle-soft-deprecated.svg}{options: alt='[Soft-deprecated]'}}}{\\strong{[Soft-deprecated]}}"
```

https://github.com/r-lib/lifecycle/blob/master/man/figures/lifecycle-soft-deprecated.svg

Reply:
I do have 1.0.0.`usethis::use_lifecycle()` doesn't save that file so I will download manually.

Reply:
It looks like I was wrong, soft-deprecated is now just a "subclass" of deprecated.
https://github.com/r-lib/lifecycle/issues/106

They superseded the soft-deprecated badge and removed the documentation for it recently
https://github.com/r-lib/lifecycle/commit/1d25526efbc37f0d47f81825220f3cf7e42db3f4

So I guess we should be using the `"deprecated"` badge, alongside the `deprecate_soft()` function.

Sorry for the confusion, I totally misunderstood 😞